### ISSUE TITLE
Support for OpenJDK 21

### ISSUE THREAD

**Issue by mabartos** [keycloak/keycloak#26152]
### Description

Blocked-by: https://github.com/keycloak/keycloak/issues/26150

JDK21 support has been included in Quarkus since the 3.5.0 version. After upgrading Quarkus, we should make Keycloak support it as well.

**Comment by vmuzikar** [comment]
Duplicate of #28517
